Luminis.AzureStorageBackup/BackupAzureStorage.cs

```diff
@@ -90,9 +90,11 @@ public async Task BackupAzureTablesToBlobStorage(IEnumerable<string> sourceTable
 
             this.logger?.LogInformation($"BackupAzureTablesToBlobStorage - From: {this.sourceAccountName}, Tables: {string.Join(", ", sourceTables)} to {targetAccountName}/{targetContainerName}");
 
+            string journalFileFolder = $@"{Environment.GetEnvironmentVariable("LocalAppData")}\{Guid.NewGuid()}";
```

Comment:
Can you put this in a temp folder instead of localAppData? That way, clean-up should be easier.

Reply:
If successful the directory is already removed, if failed you can use it for debug purposes, because it only persists on failure i don't think it needs to be in the temp folder.